src/scales.py

```diff
@@ -39,4 +39,49 @@ def set_default_locators_and_formatters(self, axis):
         axis.set_major_locator(ticker.MultipleLocator(base=numpy.pi))
 
 
+class SquareRootScale(scale.ScaleBase):
+    """Class for generating custom square root scale."""
+    name = "squareroot"
+
+    def __init__(self, axis):
+        scale.ScaleBase.__init__(self, axis)
+
```

Comment:
Is this needed? Simply removing this should result in the same init. Unless I'm missing something here.

Reply:
I've added this as this is what all examples I found online were doing, see [this for example](https://stackoverflow.com/questions/64187145/trying-to-make-a-cubed-scalebase-class-in-matplotlib-but-i-cant-get-it-working), but honestly critically looking at it I don't see either that it actually does anything other than the standard init. It's completely repetitive, removing it doesn't break anything either.

Also looking at [official documentation](https://matplotlib.org/stable/gallery/scales/custom_scale.html) gives me the impression this init is just redundant. So I am removing this.